nikola/plugin_categories.py

```diff
@@ -93,6 +93,10 @@ def __init__(self, *args, **kwargs):
         BasePlugin.__init__(self, *args, **kwargs)
         DoitCommand.__init__(self)
 
+    def __call__(self, config=None, **kwargs):
```

Comment:
amazing hack :)

Looking at yapsy you could sub-class `PluginManager.instanciateElement()` to avoid this.